nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -235,20 +228,20 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
 
   @Override
   public void renameTable(TableIdentifier from, TableIdentifier toOriginal) {
-    reference.checkMutable();
+    client.getRef().checkMutable();
 
     TableIdentifier to = NessieUtil.removeCatalogName(toOriginal, name());
 
-    IcebergTable existingFromTable = table(from);
+    IcebergTable existingFromTable = client.table(from);
     if (existingFromTable == null) {
-      throw new NoSuchTableException("table %s doesn't exists", from.name());
+      throw new NoSuchTableException("table '%s' doesn't exists", from.name());
```

Comment:
As long as we're touching these, could we use the more standard exception messages? This is usually `Table does not exist: %s`. That's nice because you don't need quotes to see when it is empty.

Reply:
sure, will update those accordingly